config/clusters/nasa-veda/common.values.yaml

```diff
@@ -68,7 +68,7 @@ basehub:
           description: Pangeo based notebook with a Python environment
           default: true
           kubespawner_override:
-            image: public.ecr.aws/nasa-veda/nasa-veda-singleuser:2024-03-07
```

Comment:
You should update the config for the GHG and ESDIS hubs as well here.

Reply:
let's wait for the esdis hub until the end of the webinar tomorrow Mar 21st at 1 pm CST

Reply:
@yuvipanda updated the PR to include the GHG and ESDIS hubs. Let's wait for a review from @ranchodeluxe as well as Mar 21st 1pm CST as per @slesaad's comment :)

Reply:
we're good to go with this!